### ISSUE TITLE
Oauth2 group causes panic in cloudflare_access_group

### ISSUE THREAD

**Issue by ideologysec** [cloudflare/terraform-provider-cloudflare#965]
I'm having the same issue as #947 with Oauth2 groups - specifically GSuite.

Terraform v0.14.7
+ provider registry.terraform.io/cloudflare/cloudflare v2.18.0

```yaml
resource "cloudflare_access_group" "example" {
  zone_id          = var.cloudflare_zone_id
  name             = "Example Group"
  include {
    gsuite {
      email                 = ["example@example.com"]
      identity_provider_id = "54791972-28fa-49c0-b3a7-0887ed38634e"
    }
  }
}
```

the resulting group is actually created successfully, but the cloudflare provider still panics:

```bash
cloudflare_access_group.example: Creating...

Error: rpc error: code = Unavailable desc = transport is closing


panic: interface conversion: interface {} is nil, not string
```

Can this be fixed for Oauth2 as well?

_Originally posted by @ideologysec in https://github.com/cloudflare/terraform-provider-cloudflare/issues/947#issuecomment-782233190_

**Comment by ideologysec** [comment]
per #947, it looks like the identity-group-id is no longer returned, so an attempt to parse a null string is causing the panic. (and a quick peek at the code seems to confirm it).

**Comment by jacobbednarz** [comment]
The example you're showing above was raised via #935 already addressed in #940 (scheduled for the 2.19.0 release). Can you please build the provider per the README instructions and confirm whether or not it works for you?
